getting-started/what-you-need.md

```diff
@@ -11,197 +11,248 @@ position: 1
 
 # Typical Workflow for Using the UI for Blazor Components
 
-This article describes the steps in the typical workflow for using the Telerik UI for Blazor components&mdash;getting the Telerik UI for Blazor components and configuring your project to use them.
+This article describes the required steps to use the Telerik UI for Blazor components in any kind of Blazor application and any .NET version. The content below describes how to obtain the Telerik UI for Blazor components and configure your project to use them.
```

Comment:
This Typical Workflow article is a bit of a legacy. Until present, it pretty much repeated the content of the First Steps articles. What I did now is:

* Make the First Steps articles simpler.
* Make the Typical Workflow article complex.

The changes were synced with Dancho in advance.

The Typical Workflow article is "typical" in the sense that it describes all required getting started steps, but provides alternatives and more in-depth information. On the other hand, the First Steps articles only give the easiest "happy path".

If we remove the Typical Workflow article, we will be forced to create new content elsewhere, which provides the same information. I believe this will make the documentation structure even more complex and the information will be harder to find.

Reply:
Thanks for the additional context, that really helps. However, I can't help but feel that there's been some misunderstanding along the way. Nowhere have I suggested removing the page. I believe it has its value but there's additional work needed for it to shine through and become truly helpful to the reader. 

Here's a more detailed breakdown of the work that I believe could improve this page and why.

* **Problem:** The title is misleading. It advertises the page as "typical workflow", but instead provides tons of concepts and references and doesn't really follow a "flow".
**Suggested solution:** Consider reworking the logical structure of the page to resemble a workflow. As noted in my initial comment, this might be "install -> import -> register -> use components in a view", but it might be something else, based on your expertise. What we're looking for are action-based section that achieve certain goals.
Alternatively, rename the page to something more fitting its current state - this might be the easiest change and might address some of the other issues as well.
* **Problem:** This page is the first one in your Getting Started section. This is the first thing people will click and go through. As pointed in the page itself, this is not a beginner page. As pointed in your reply, this is a more in-depth page. Instead of helping users get started with Blazor, it would overload them with information and would point them to additional resources, which would prolong their journey or hinder it altogether.
**Suggested solution:** Change the page's position within the section. You might also want to consider reworking your Getting Started section as a separate effort or introducing an "orientation" page that would split people in the right reading journey for their case (so that you don't need to take up most of the first reading screen on this page with the callout for other pages).
* **Problem:** The page is overloaded with various colorful callouts. This draws attention away from the actual text. Usually, callouts are should be used sparingly and only to create meaningful emphasis.
**Suggested solution:** Consider incorporating some of the callouts within the regular text, where it makes sense, to limit the distractions. Alternatively, you can rework them in a section of important considerations. 
* **Problem:** The page is overloaded with information. This might be due to the fact, that based on your reply, it seems this page now hosts information that couldn't live elsewhere.
**Suggested solution:** Consider investing some time in smart editing (prune whatever needs to be pruned). I am not entirely sure that this is possible, so you might want to ignore this point.

As always, this is your domain - you know your users and your subject matter. So even though relevant in the domain of technical writing, my recommendations might need proper adjustment for your user base. If you find any of these helpful, feel free to plan the necessary changes as a separate effort. You are also always welcome to reach out and discuss the details in person.